.github/ISSUE_TEMPLATE/RFC.md

```diff
@@ -0,0 +1,70 @@
+---
+name: RFC
+about: Propose new feature(s) or major changes to the component
```

Comment:
adding `label: rfc` would always add the `RFC` label (when created) during issue/RFC creation.


```bash
for organization in laminas laminas-api-tools mezzio; do echo $organization; for package in $(gh api -X GET /orgs/$organization/repos | jq -r '.[] .name'); do echo $package; gh api /repos/$organization/$package/labels -X POST -f name="RFC" -f color="#f7e101" -f description="Request for Comments. Contains proposals for changes in one or multiple packages."; done ; done
```

Reply:
I would prefer having the date in case we are importing RFCs from e.g. discourse. But yes, if not imported, we should use the issue creation date instead.
Wondering if its possible to create issues via API and passign the creation date (which might be in the past). If so, I'd prefer using that instead of passing the date via issue text.

Reply:
if we are importing - we are not using this template
we cannot set created date in the past, we tried during the migration from ZF